File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.js
function solution(n) {
    var answer = "";
    let s = String(n)
    s = s.split("").sort((a,b) => b-a)
    
    for(let i = 0; i < s.length; i++){
        answer += s[i]
    }
    
    answer = Number(answer)
    
    return answer;
}

File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.py
def solution(n):
    answer = 0
    
    list_n = list(map(int, str(n)))
    list_n.sort(reverse = False)
    
    for i in range(len(list_n)):
        print(i, list_n[i])
        answer += (list_n[i])*10**(i)    
    return answer